src/test/java/FooterTest.java

```diff
@@ -0,0 +1,94 @@
+import components.Footer;
```

Comment:
One more test is missing -> copyright section.
`Copyright © 2013-present Magento, Inc. All rights reserved.`